docs/networks/l16-testnet/start-node.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 2
+---
+
+# Start your L16 node
+
+## Create a directory
+
+ ```
+ mkdir lukso-l16-testnet 
+ ```
+ ### navigate to it in your terminal by using the `cd` command
+ ```bash
+ cd lukso-l16-testnet
+ ```
```

Comment:
We separated those commands to let people learn what they are actually doing. Most validators are no devs at all. I think Network team does want to have it this clear since people are running and installing commands in home directory. But I will leave this question for @NandoTheessen

Reply:
Ok makes sense, let's keep it like this them. thks